### PR TITLE
UX: Cleanup mobile topic footer area

### PR DESCRIPTION
**Pull request by chapoi**
Any "uneven" spacing is caused by empty divs, courtesy of Ember. I'm opting to tolerate these for now, knowing they'll soon(ish) be gone, rather than doing only half a rework, and needing a revision later.

## Desktop
CleanShot 2024-10-22 at 19 29 49@2x

### With typing indicator
CleanShot 2024-10-23 at 14 17 04@2x

### Check with all plugin outlets in use
CleanShot 2024-10-23 at 15 59 43@2x

## Mobile
CleanShot 2024-10-22 at 19 49 48@2x

### Narrow
CleanShot 2024-10-22 at 19 50 23@2x

### Wide
CleanShot 2024-10-22 at 19 51 03@2x

### With typing indicator
CleanShot 2024-10-22 at 19 52 37@2x

### With ToC
Needs a little adjustment to ToC button, but will address that in ToC component
CleanShot 2024-10-23 at 14 26 34@2x

### Check with all plugin outlets in use (not ideal obv)
CleanShot 2024-10-23 at 16 00 21@2x

### CODE REVIEW

File: app/assets/javascripts/discourse/app/components/topic-footer-buttons.hbs [hunk 1]
```diff
@@ -1,111 +1,187 @@
...
+        @appendReason={{false}}
+      />
+
+      {{#if this.showNotificationsButton}}
+        {{#if this.showNotificationUserTip}}
+          <UserTip
+            @id="topic_notification_levels"
+            @triggerSelector=".notifications-button"
+            @titleText={{i18n "user_tips.topic_notification_levels.title"}}
+            @contentText={{i18n "user_tips.topic_notification_levels.content"}}
```
Comment (chapoi) [inline app/assets/javascripts/discourse/app/components/topic-footer-buttons.hbs:129]: Haven't been able to test this properly, if it's still works well

**Comment by chapoi** [comment]
Me trying to merge things broke stuff, so I reverted back to my last working version. I'll someone more apt with merging handle it

**Comment by ZogStriP** [comment]
I'm having a look 👀

**Comment by ZogStriP** [comment]
Superseded by #30132

**Pull request closed without merging**